Gruntfile.js

```diff
@@ -62,6 +62,9 @@ module.exports = function(grunt) {
         files: [{
```

Comment:
I don't remember exactly where or what the problem is, but fix the Grunt weirdness when running `grunt` with no arguments